glyphs2fontir/src/source.rs

```diff
@@ -27,15 +27,21 @@ impl GlyphsIrSource {
 }
 
 struct PlistCache {
-    _global_metadata: StateSet,
+    global_metadata: StateSet,
     _root_dict: HashMap<String, Plist>,
```

Comment:
underscore prefix is unnecessary for a private-by-default struct field, no?

Reply:
I would have sworn it was warning about it, let me double-check

Reply:
global_metadata is now used, I tried with root_dict and it did complain:

```
warning: field `root_dict` is never read
  --> glyphs2fontir/src/source.rs:31:5
   |
29 | struct PlistCache {
   |        ---------- field in this struct
30 |     global_metadata: StateSet,
31 |     root_dict: HashMap<String, Plist>,
   |     ^^^^^^^^^
   |
   = note: `#[warn(dead_code)]` on by default
```